projects/admin/src/app/record/operation-logs/operation-logs.component.html

```diff
@@ -25,46 +25,49 @@ <h5 class="modal-title" translate>Operation history</h5>
       </div>
       <div class="modal-body overflow-auto overflow-max-size">
         <ng-container *ngIf="loadedRecord; else loaded">
-          <ng-container *ngIf="createRecord">
+          <ng-container *ngIf="createRecord || records.length > 0; else noRecord">
             <ng-container *ngIf="createRecord">
-              <ul class="list-unstyled mx-2 mb-3">
-                <li>
+              <ng-container *ngIf="createRecord">
+                <ul class="list-unstyled mx-2 mb-3">
+                  <li>
+                    <div class="row col">
```

Comment:
this is not a good idea to put "row col" in the same element, put the "row" on the `li`. Can you check for other cases?